boot/systems.go

```diff
@@ -94,6 +104,8 @@ func SetTryRecoverySystem(dev Device, systemLabel string) (err error) {
 		return fmt.Errorf("internal error: recovery systems can only be used on UC20")
 	}
 
+	model := dev.Model()
```

Comment:
this will  be resealed-to model because the caller passes the remodel ctx for dev?

Reply:
if I understand correctly, the doc comment for the function should say something like:

if dev refers to a different model, typically because it's a remodel context,  TryModel on modeenv will be set to it before resealing

Reply:
i've updated both doc comments and added a note in the code